webapp/src/components/loading/index.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import './loading.scss';
+
+const Loading = () => {
+    return (
+        <div
+            className='loading-screen'
+            style={{position: 'relative'}}
```

Comment:
This PR addas a scss file and also does some style here. I'm thinking we should use just use `style` and avoid adding the small scss file

Reply:
@mickmister Updated the file with inline styling and deleted the scss file. Please have a look.